wp-content/themes/mrb-bct/page-client.php

```diff
@@ -0,0 +1,183 @@
+<?php 
+session_start();
+
+/* Template Name: Client Listing */ 
+
+$user =wp_get_current_user();
+$feed = file_get_contents(ABSPATH . 'fixtures/jobs.xml');
+$mrb_job_xml = simplexml_load_string($feed);
+if(!isset($_POST['job'])||''==$_POST['job'])
```

Comment:
I appreciate you're checking for a `job` in the post data, but is there a less verbose way of achieving this using built-in PHP functions?